manifests/server/backup.pp

```diff
@@ -67,6 +67,10 @@
 #   Specifies an array of optional arguments which should be passed through to the backup tool. (Supported by the xtrabackup and mysqldump providers.)
 # @param install_cron
 #   Manage installation of cron package
+# @param compression_command
+#   Configure the command used to compress the backup (when using the mysqldump provider)
```

Comment:
Could you add a note to this stating the lack of package installation?

Reply:
Absolutely. Added.